transforms/angle-brackets/__testfixtures__/html-tags.input.hbs

```diff
@@ -0,0 +1,17 @@
+<input 
+	type='text'
+	value={{userValue}}
+	oninput={{action 'change' value='target.value'}}
+	class="{{if invalid 'invalid-input'}}"/>
+
+<textarea 
+	value={{userValue}}
+	oninput={{action 'change' value='target.value'}}
+	class="{{if invalid 'invalid-input'}}">HI</textarea>
+
+<textarea 
+	value={{userValue}}
+	oninput={{action (action 'change') value='target.value'}}
+	class="{{if invalid 'invalid-input'}}">HI</textarea>
+
+<div onclick="{{action "clickMe"}}"></div>
```

Comment:
I don't believe that this should be quoted

Reply:
COPYPASTA.  Thanks @rwjblue